### ISSUE TITLE
Improve accessibility action labels for MasterTimelineCell in iOS

### ISSUE THREAD

**Issue by babbage** [Ranchero-Software/NetNewsWire#3072]
In NetNewsWire for iOS, MasterTimelineCells correctly implement accessibility action labels for the functions that can be accessed via swift actions. One of these actions is to mark the article as either read or unread, depending on the current status of the article. The current labels for these actions are "Read" and "Unread". However, VoiceOver voices "Read" to rhyme with _need_ not _bread_. This is actively misleading for a VoiceOver user, because the action "read" (rhymes with _need_) would be expected to open the article for reading... which is after all the primary function of NetNewsWire. Instead, this action marks the article as _having been read_.

Proposed change: rename these actions to "Mark as read" and "Mark as unread". VoiceOver labels should be as short as possible _while being clear_. The current labels are _too short_ to be clear. "Read" turns out to be actively misleading, due to the VoiceOver pronunciation of this ambiguous word, and "Unread" is just less clear as an action than "Mark as unread".

While this change could be trivially made in the code, I propose to make a pull request that includes unit testing to ensure that there is not a future regression in this fix—since it would be easy for a person in the future to unwittingly think that changing these action labels back to "Read" and "Unread" was fine.

A screen recording demonstrates this issue. See the bullet points that follow for further explanation: 

https://user-images.githubusercontent.com/376937/115975078-dbc95300-a5b5-11eb-83fc-89f5c02ea578.mp4

The swipe actions taken by the user in the video:
0) VoiceOver was activated via the usual accessibility settings/shortcuts (before this recording starts)

1) Swipe to the right a series of times, each time stepping through one interface element, until the first tableview cell is selected

2) On reaching the cell, VoiceOver reads out the defined cell content, ending with "Actions available", which indicates there are actions the user can access via this cell.

3) Swipe down a series of time, each one voicing one action available.

4) On the action "Read", double tap on the screen to activate that action.

5) The cell is marked as Read, and the cell contents are voiced again.  (Somewhat confusingly, after initiating this action VoiceOver then reads out the cell VoiceOver content again, including an article preview from the cell again—which is longer than the text displayed in writing on the cell and thus even potentially gives the initial impression that the article _is_ being read out.

**Comment by brentsimmons** [comment]
Great! Please submit the PR to the ios-candidate branch. Thanks!
